nix/cells/automation/pipelines.nix

```diff
@@ -0,0 +1,73 @@
+{ cell
+, inputs
+}:
+let
+  inherit (inputs.nixpkgs) system;
+
+  common =
+    { config
+    , ...
+    }: {
+      preset = {
+        nix.enable = true;
+
+        github-ci = {
+          # Tullia tasks can run locally or on Cicero.
+          # When no facts are present we know that we are running locally and vice versa.
+          # When running locally, the current directory is already bind-mounted
+          # into the container, so we don't need to fetch the source from GitHub
+          # and we don't want to report a GitHub status.
+          enable = config.actionRun.facts != { };
+          repo = "input-output-hk/plutus";
+        };
+      };
+
+      memory = 1024 * 8;
+      nomad.resources.cpu = 10000;
+    };
+in
+{
+  ci = { config, lib, ... }: {
+    imports = [ common ];
+
+    preset.github-ci.sha =
+      config.preset.github-ci.lib.readRevision inputs.cells.cloud.library.actions.ci.input null;
+
+    command.text = ''
+      nix build -L .#${lib.escapeShellArg system}.automation.ciJobs.required
+    '';
+  };
+
+  publish-unraveling-recursion-paper = { config, pkgs, lib, ... }: {
+    imports = [ common ];
+
+    preset.github-ci.sha =
+      config.preset.github-ci.lib.readRevision inputs.cells.cloud.library.actions.paper.input null;
+
+    command.text = ''
+      set -x
+
+      nix build -L .#${lib.escapeShellArg system}.plutus.packages.unraveling-recursion-paper
```

Comment:
Here is the full list of papers.
Note that we can do without the `${lib.escapeShellArg system}`
```



nix build .#plutus-report
nix build .#plutus-core-spec
nix build .#extended-utxo-spec

nix build .#unraveling-recursion-paper
nix build .#system-f-fin-agda-paper
nix build .#eutxo-paper
nix build .#utxoma-paper
nix build .#eutxoma-paper
```

Reply:
> Note that we can do without the ${lib.escapeShellArg system}

That's because the flake CLI magically inserts it for you, though? I'd rather be explicit, tbh.

Reply:
report and core spec are the exact same:

```
❯ diff ./nix/cells/plutus/packages/plutus-*
4,5c4,5
<   name = "plutus-core-spec";
<   description = "Plutus core specification";
---
>   name = "plutus-report";
>   description = "plutus report";
```

Reply:
Well, that's a bug.

However, I mostly want a parametric version of this. Then you can leave it to us to make sure it's building the right set of artifacts :)